admin/app/dfp/DfpApi.scala

```diff
@@ -23,9 +20,13 @@ object DfpApi extends Logging {
         })
     })
 
-    DfpLineItems(lineItems
+    val validatedLineItems = lineItems
```

Comment:
Minor point but there is varying use of `item` vs `lineitem`, maybe rename this to `validatedItems` for consistency...